explainer.md

```diff
@@ -1,8 +1,45 @@
 # Badging API Explainer
 
-Author: Matt Giuca <mgiuca@chromium.org>
-
-Date: 2018-06-26
+Author: Matt Giuca <mgiuca@chromium.org><br>
+Author: Jay Harris <harrisjay@chromium.org><br>
+Author: Marcos Cáceres <marcosc@mozilla.org>
```

Comment:
Is this your preferred email @marcoscaceres ? (assuming you want to be added :P)